backend/app/db/schema.py

```
@@ -0,0 +1,37 @@
+
+    for label, property_name in indexes:
+        try:
+            query = f"CREATE INDEX FOR (n:{label}) ON (n.{property_name})"
+            graph.query(query)
+            logger.info(f"Created index on {label}({property_name})")
+        except Exception as e:
+            # FalkorDB will raise an error if index already exists. We can safely ignore it.
+            if "already exists" in str(e).lower() or "index already exists" in str(e).lower():
+                logger.debug(f"Index on {label}({property_name}) already exists.")
+            else:
+                logger.warning(f"Error creating index on {label}({property_name}): {e}")
```

Ruff (0.15.11)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)